### ISSUE TITLE
extractorapp - actions - org.georchestra.extractorapp.ws.extractor.wcs.ScaleUtilsTest fails

### ISSUE THREAD

**Issue by fvanderbiest** [georchestra/georchestra#2925]
https://github.com/georchestra/georchestra/commit/e79986dd52353953f6677418679c8be497afe652/checks?check_suite_id=437192060
```
[INFO] Running org.georchestra.extractorapp.ws.extractor.wcs.ScaleUtilsTest
[ERROR] Tests run: 2, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.265 s <<< FAILURE! - in org.georchestra.extractorapp.ws.extractor.wcs.ScaleUtilsTest
[ERROR] testGetUnit(org.georchestra.extractorapp.ws.extractor.wcs.ScaleUtilsTest)  Time elapsed: 1.043 s  <<< FAILURE!
org.junit.ComparisonFailure: expected:<[°]> but was:<[deg]>
	at org.georchestra.extractorapp.ws.extractor.wcs.ScaleUtilsTest.testGetUnit(ScaleUtilsTest.java:19)
```

**Comment by RemiDesgrange** [comment]
I think that in bug report like this you should put the targeted branch/or commit that you target WDYT ?

**Comment by fvanderbiest** [comment]
The commit hash is in the URL ... or I do not understand your request...

**Comment by RemiDesgrange** [comment]
> The commit hash is in the URL ... or I do not understand your request...

:tired_face:  tired

**Comment by fvanderbiest** [comment]
Fixed with #2937
